code/iaas/service-discovery/api/src/main/java/io/cattle/platform/servicediscovery/api/resource/ServiceDiscoveryConfigItem.java

```diff
@@ -58,6 +58,9 @@
 
     public static final ServiceDiscoveryConfigItem LABELS = new ServiceDiscoveryConfigItem(
             InstanceConstants.FIELD_LABELS, InstanceConstants.FIELD_LABELS);
+    public static final ServiceDiscoveryConfigItem MEMSWAPLIMIT = new ServiceDiscoveryConfigItem("memorySwap", "memswap_limit");
```

Comment:
Moving the params to DockerInstanceConstants , but the "_" conversion is probably not applicable here.